cas-offinder.cpp

```diff
@@ -323,7 +323,7 @@ void Cas_OFFinder::writeHeaders(const char* outfilename) {
 	} else {
 		fo = new ofstream(outfilename, ios::out | ios::trunc);
 	}
-	(*fo) << "#Id\tBulge type\tcrRNA\tDNA\tChromosome\tLocation\tDirection\tMismatches\tBulge Size" << endl;
+	(*fo) << "Id\tBulge type\tcrRNA\tDNA\tChromosome\tLocation\tDirection\tMismatches\tBulge Size" << endl;
```

Comment:
Hi, thank you for the suggestions!
Here the prefix `#` is intended to be similar to the VCF file format (https://samtools.github.io/hts-specs/VCFv4.2.pdf).

Reply:
Ok, thanks for mentioning, I'll revert those changes tomorrow.

FWIW, since it's not a numerical ID, and the README elsewhere uses the term `Label` (nicer than `Id`, I agree), then maybe the output column name should also be `Label`, i.e. `#Label\tBulge Type\t...` ?  One or the other, not both terms, is what I mean here.

Aside, why is VCF similarity helpful for cas-offinder output?  I am curious because I do most processing in either shell tools or a programming language.  `#` lines look like comments to me. :)  If being similar to VCF, perhaps the other `##` lines from that standard should be included if relevant?

Reply:
Yes. See this: https://github.com/snugel/cas-offinder/issues/39

Also, I consider providing the version of Cas-OFFinder too.

Reply:
Maybe this is an English question: Does ID sound like it is a number? ID can be string, isn't it? For example, we use usernames, a.k.a ID, as strings...

Thanks for the suggestion, but for me, the term `label` doesn't sound like it is a unique one per target.
In other packages, when we call unique strings, Pandas/SQL uses `index`, Python in general uses `key`, but I thought `ID` sounds better than these two...

Reply:
Re: #39 -- I did not see that, great! :)

Re: label vs. id -- Your comments are good.  I guess it will depend on personal experience to some extent.  I agree `label` seems less likely to be unique than `id`.  Common usage of `label` is probably more like `tag`, which would definitely not be unique and probably be usage for a grouping or classification.

I was mostly concerned with consistency in terminology, so I just changed `label` to `ID` in the README. :)